app/app/settings.py

```diff
@@ -173,6 +174,21 @@
 
 LOGGING_FOLDER_DEFAULT = os.path.abspath(os.path.join("/logging/"))
 
+# Internationalization
+
+USE_I18N = True
+
+LANGUAGES = [
+    ("en", "English"),
+    ("af", "Afrikaans"),
+]
```

Comment:
For maintainability, please keep it alphabetically sorted by language code.

Reply:
will do